docs(concepts): describe virtual instances (#4535)
zitadel/zitadel: docs/docs/concepts/structure/instance.mdx

title: Instance
---

## Instance Structure

An instance is the top node in ZITADEL's data hierarchy.
Within an instance all the default [settings](/docs/concepts/structure/policies),
such as branding, login policy, password policy, etc. for the system can be configured.

> One instance normally runs on one domain and has one issuer (e.g login.customer.com).

Reviewer: ```suggestion
One instance normally runs on one domain and represents one issuer (e.g login.customer.com).
```

Reviewer: ZITADEL is able to respond on multiple domains, but only one can be the main domain which is used to send mails and other things like the console access.

Reviewer: I think it is worth mentioning what I stated above

Author: 😄 ok

Reviewer: I thought so 😁

One instance can contain multiple [organizations](/docs/concepts/structure/organizations),

> which in turn can represent the own company (e.g. departments) or business customers.

Reviewer: ```suggestion
which in turn can represent your own company (e.g. departments), your business customers or a consumer organization.
```

Read more about how to configure your instance in our [instance guide](/docs/guides/manage/console/instance-settings).

## Multiple Virtual Instances

> ZITADEL also has the concept of virtual instances.

Reviewer: ```suggestion
ZITADEL has the concept of virtual instances.
```

When installing ZITADEL from scratch, one instance is always automatically created for you.
Nevertheless, you can add more virtual instances via the [system API](/docs/apis/proto/system#addinstance).

> This is especially useful if you have business customers, which in turn have their business customers with self service and custom domain demands.

Reviewer: ```suggestion
This is useful if you have business customers, which in turn have their business customers with self service and custom domain demands.
```

> By providing full-blown virtual ZITADEL instances, your customers have all the customization options available in ZITADEL.

Reviewer: ```suggestion
By providing a virtual ZITADEL instances, your customers have all the customization options available in ZITADEL.
```
